blockly-core/core/ui/contract_editor/contract_editor.js

```diff
@@ -152,37 +168,146 @@ Blockly.ContractEditor.prototype.create_ = function() {
   );
 
   this.hiddenExampleBlocks_ = [];
-  /** @type {Blockly.SvgTextButton} */
-  this.addExampleButton = new Blockly.SvgTextButton(
-    canvasToDrawOn,
-    "Add Example", // TODO(bjordan): i18n
-    this.addNewExampleBlock_.bind(this)
-  );
+  this.exampleAreaDiv = goog.dom.createDom('div', 'exampleAreaDiv');
+  this.addExampleButton = goog.dom.createDom('button', 'exampleAreaButton btn');
+  this.addExampleButton.innerHTML = "Add Example";
+  Blockly.bindEvent_(this.addExampleButton, 'click', this, this.addNewExampleBlock_);
+  goog.dom.append(this.exampleAreaDiv, this.addExampleButton);
+  this.exampleAreaDiv.style.display = 'block';
+  this.exampleAreaDiv.style.position = 'absolute';
+  goog.dom.insertChildAt(this.container_, this.exampleAreaDiv, 0);
+
+  var callText = goog.dom.createDom('div', 'callText');
+  callText.innerHTML = "Call";
+  goog.dom.appendChild(this.exampleAreaDiv, callText);
+
+  var resultText = goog.dom.createDom('div', 'callText');
+  resultText.innerHTML = "Result";
+  goog.dom.appendChild(this.exampleAreaDiv, resultText);
+
+  var examplesTableGroup = Blockly.createSvgElement('g', {}, canvasToDrawOn);
+
+  var topHorizontalLine = Blockly.createSvgElement('rect', {
+    'fill': '#000'
+  }, examplesTableGroup);
+  topHorizontalLine.setAttribute('height', 2.0);
+
+  // TODO(bjordan): get horizontal line #X helper, lay out below examples
+
+  var verticalMidline = Blockly.createSvgElement('rect', {
+    'fill': '#000'
+  }, examplesTableGroup);
+  verticalMidline.setAttribute('width', 2.0);
+
+  /**
+   * @type {Blockly.ExampleView[]}
+   * @private
+   */
+  this.exampleViews_ = [];
 
   this.examplesSectionView_ = new Blockly.ContractEditorSectionView(
     canvasToDrawOn, {
       sectionNumber: 2,
+      headerHeight: HEADER_HEIGHT,
       headerText: "Examples", // TODO(bjordan): i18n
       placeContentCallback: goog.bind(function (currentY) {
+        var maxWidth = +this.exampleBlocks.reduce(function (previousMax, block) {
```

Comment:
I wouldnt think we need the plus here, as we start with a number (0) and always add a number (Math.max)